app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -350,6 +351,9 @@ protected void onCreate(Bundle savedInstanceState) {
             return;
         }
 
+        // show app update dialog
```

Comment:
Can we have a Don't ask again button here, as if someone feels comfortable in this app, it might be spammy to show them this every time?

Reply:
I also though of doing it, but then thought of not investing much time on this.
But ya! now updated 👍